app/src/main/java/com/vultisig/wallet/data/api/BlowfishApi.kt

```diff
@@ -59,7 +59,7 @@ internal val Chain.blowfishChainName: String?
         Chain.ethereum -> "ethereum"
         Chain.polygon -> "polygon"
         Chain.avalanche -> "avalanche"
-        Chain.arbitrum -> "arbitrum"
+        Chain.Arbitrum -> "arbitrum"
```

Comment:
I want to update others in the incoming PR because the changes are too extensive. 
So for now, I will revert the changing of the chain name and change them in the next PR. 🛠️